.gitlab-ci.hpc-spack.yml

```diff
@@ -0,0 +1,49 @@
+include:
+  - project: hpc/gitlab-pipelines
+    file: spack-build-components.gitlab-ci.yml
+
+spack_setup:
+  extends: .spack_setup_ccache
+  variables:
+   SALLOC_ACCOUNT: proj3
+   SBATCH_ACCOUNT: proj3
+   SLURM_ACCOUNT: proj3
+   bb5_account: proj3
+   SPACK_PACKAGE_NAME: brayns
+   SPACK_PACKAGE_COMPILER: gcc
+   SPACK_PACKAGE_SPEC: +brion ^ospray@1.8.5%intel
+   DATADIR: /tmp 
+
+
+brayns-spack-build:
+  extends: .spack_build
+  stage: test
+  variables:
```

Comment:
These `variable` blocks could still be on the file level? Did that not work? Especially since they are repeated 3 times :)

Reply:
Thanks!, Just made them file level to test it again (didnt really test them the previous time)
For the .ctest job, we are adding CTests in brayns, but seems like spack cannot find them. Is there any special way we have to set them up, or any variable I am not declaring for it?